src/ctype/CType.ts

```diff
@@ -16,8 +16,39 @@ import ICType from '../types/CType'
 import Identity from '../identity/Identity'
 import { getOwner, store } from './CType.chain'
 import TxStatus from '../blockchain/TxStatus'
+
 import IClaim from '../types/Claim'
 
+export function compressCTypeSchema(cTypeSchema: ICType['schema']): any[] {
+  return [
```

Comment:
this looks error prone. Perhaps some thing like this:

```js
obj = {a: 1, c: 132, xyx: 125, fs: 12, rust: "<3"}
Object.keys(obj)
  .map((v, i)=>[v, obj[v]])
  .sort((a, b) => a[0].localeCompare(b[0]))
  .map(a => a[1])
```

Reply:
As previously mentioned, we are going to use the jsonabc and yes it is a problem :D